src/local/delete.go

```diff
@@ -69,32 +69,35 @@ func (m *Manager) DeleteEmptyDirectories(trackedPaths []string) error {
 			return nil
 		}
 
-		// Skip ignored
-		if utils.IsIgnoredDir(path, d) {
-			return filepath.SkipDir
-		}
+		// Stop on ignored dir
+		isIgnoredDir := utils.IsIgnoredDir(path, d)
 
 		// Found a directory -> store path
-		if d.IsDir() {
+		if !isIgnoredDir && d.IsDir() {
 			emptyDirs.Set(path+string(os.PathSeparator), true)
 			return nil
 		}
 
-		// Is file -> all parent dirs are not empty
+		// Found file/ignored dir -> all parent dirs are not empty
 		for _, dir := range emptyDirs.Keys() {
 			if strings.HasPrefix(path, dir) {
 				emptyDirs.Delete(dir)
 			}
 		}
 
+		// Skip sub-directories
+		if isIgnoredDir {
+			return filepath.SkipDir
```

Comment:
Do skryteho adresara nepotrebujeme vstupovat, ... nadradeny adresar zme oznacili ako NIE-prazdny, a to je vsetko.
Obsah skryteho adresara nas nezaujima.